### DISCUSSION TITLE
你打包后的安装包是怎么缩小的？

### DISCUSSION THREAD

**Discussion by shinexoh** [lollipopkit/flutter_server_box#49] [General]
我看你Releases打包后的安装包里打包了v7a、v8a、x86等架构，按理来说你这安装包起码20多mb，想问问你用的什么优化方法做到才10mb的？

**Comment by wuge0** [comment]


**Reply by shinexoh** [reply]
但是我用as的虚拟机安装了一下，虽然可以安装但是打开就闪退停止运行，我看你安装包的lib里就v8a架构是完整的so库文件，其他的好像并没有作用:)

**Comment by lollipopkit** [comment]
app主体只打包了arm64 :)

其他架构是因为有个依赖(rupgrade)

**Reply by shinexoh** [reply]
行吧 :)
